geps/gep-713/index.md

```diff
@@ -1,1157 +1,1082 @@
-    more focussed GEP-2648 and GEP-2649.
-    Some options under discussion there may make the distinction between Direct
-    and Inherited Policies moot, which would require a rework.
-
-This GEP aims to standardize terminology and processes around using one Kubernetes
-object to modify the functions of one or more other objects.
-
-This GEP defines some terms, firstly: _Metaresource_.
-
-A Kubernetes object that _augments_ the behavior of an object
-in a standard way is called a _Metaresource_.
-
-This document proposes controlling the creation of configuration in the underlying
-Gateway data plane using two types of Policy Attachment.
-A "Policy Attachment" is a specific type of _metaresource_ that can affect specific
-settings across either one object (this is "Direct Policy Attachment"), or objects
-in a hierarchy (this is "Inherited Policy Attachment").
-
-Individual policy APIs:
-
-- MUST be their own CRDs (e.g. `TimeoutPolicy`, `RetryPolicy` etc),
-- MUST include both `spec` and `status` stanzas
-- MUST have the `status` stanza include a `conditions` section using the standard
-  upstream Condition type
-- MAY be included in the Gateway API group and installation or be defined by
-  implementations
-- MUST include a common `TargetRef` struct in their specification to identify
-  how and where to apply that policy.
-- MAY affect more objects than the object specified in the `targetRef`. In this
-  case, the Policy is an Inherited Policy. A common way to do this is to include
-  either a `defaults` section, an `overrides` section, or both.
-- Policy objects that affect _only_ the object specified in the `targetRef` are
-  Direct Attached Policies (or more simply, Direct Policies.)
-
-The biggest difference between the two types of Policy is that Direct Attached
-Policies are a strict subset of Policy objects with criteria designed to make
-it _much_ easier to understand the state of the system, and so are simpler to
-use and can use a more simple `status` design.
-
-However, Inherited Policies, because of the nature of the useful feature of having
-settings cascade across multiple objects in a hierarchy, require knowledge of
-more resources, and are consequently harder to understand and require a more
-complex status design.
-
-Splitting these two design patterns apart into separate GEPs is intended to
-allow proceeding with stabilizing the simpler (Direct) case while we work on
-solving the status problem for the more complex (Inherited) case.
-
-Direct Attached Policies are further specified in the addendum GEP GEP-2648,
-Direct Policy Attachment.
-
-Inherited Policies are further specified in the addendum GEP-2649, Inherited
-Policy Attachment. GEP-2649 also describes a set of expected behaviors
-for how settings can flow across a defined hierarchy.
+This GEP aims to standardize terminology and processes around using one Kubernetes object to modify the functions of one or more other objects.
```

Comment:
```suggestion
This GEP aims to standardize terminology and processes around "metaresources", e.g. using one Kubernetes object to modify the functions of one or more other objects.
```

Reply:
Done, with the only difference that I used _id est_ instead of _exempli gratia_ 🙂